presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -38,6 +38,7 @@
     private Set<String> jdbcTypesMappedToVarchar = ImmutableSet.of();
     private UnsupportedTypeHandling unsupportedTypeHandling = UnsupportedTypeHandling.IGNORE;
     private int decimalDefaultScale = 16;
+    private boolean decimalRoundingEnable;
```

Comment:
- I would rename it to `unspecifiedDecimalEnabled`. WDYT? It has nothing to rounding as rounding could be disabled with `roundingMode`. It is about mapping decimal that does not fixed precision and scale set.
- please move it one line up. Before `decimalDefaultScale`

Reply:
Make sense. `unspecifiedDecimalEnabled` is much accurate name.